kythe/cxx/indexer/cxx/KytheGraphObserver.h

```diff
@@ -131,24 +131,35 @@ class KytheClaimToken : public GraphObserver::ClaimToken {
   bool language_independent_ = false;
 };
 
+struct KytheGraphObserverOptions {
+  // The custom build_config, if any, to apply to anchor nodes.
+  std::string build_config = "";
+  // The default corpus to use for nodes which would otherwise have an empty
+  // corpus.
+  std::string default_corpus = "";
+};
+
 /// \brief Records details in the form of Kythe nodes and edges about elements
 /// discovered during indexing to the provided `KytheGraphRecorder`.
 class KytheGraphObserver : public GraphObserver {
  public:
+  // Nested classes in C++ are obnoxious, so Clang won't like the default
+  // construction below if `Options` is a nested class.
```

Comment:
Sorry, I just meant this comment for you.  No need to include it in the code :-)

Reply:
sounds good :)